components/hab/src/main.rs

```diff
@@ -1363,11 +1363,13 @@ fn install_sources_from_matches(matches: &ArgMatches) -> Result<Vec<InstallSourc
 }
 
 fn excludes_from_matches(matches: &ArgMatches) -> Vec<PackageIdent> {
-    matches
-        .values_of("EXCLUDE")
-        .unwrap() // Required via clap
```

Comment:
I think a simpler fix would be changing the `unwrap` to `unwrap_or_default`.

Reply:
You're a master of the clap API :)  Done.

Reply:
FYI, that's not clap-specific, rather a feature of [`Option`](http://doc.rust-lang.org/1.30.0/core/option/enum.Option.html#method.unwrap_or_default).